### ISSUE TITLE
Meta Fine-Tuning Neural Language Models for Multi-Domain Text Mining [Wang et al., EMNLP-2020]

### ISSUE THREAD

**Issue by siida36** [siida36/NLP-survey#1]
## Paper

https://www.aclweb.org/anthology/2020.emnlp-main.250/

## Summary

タスクに特化した学習セットを用いて事前学習済みモデルを Fine-tuning することは、様々なNLPタスクにおいて有効である。しかし、事前学習済みモデルから直接パラメータを初期化しているため、異なるドメインの類似したNLPタスクの学習プロセスがどのように相関し、相互に強化されているかを無視している。

本論文では、 Meta Fine-tuning（MFT）と呼ばれる効果的な学習方法を提案している。MFT は、すべてのデータセットに対して単純にマルチタスク学習を行うのではなく、様々なドメインの典型的なインスタンスからのみ学習を行うことで、伝達性の高い知識を獲得する。さらに、 domain corruption loss function を最適化することで、言語モデルがドメインに依存しない表現をエンコードするように促す。 MFT によってモデルはより優れたパラメータの初期化が行われ、また高い汎化能力を持つようになり、各ドメインに合わせて Fine-tuning することが可能となる。

いくつかの Multi-domain text mining タスクを解決するため、 BERT に MFT を実装して実験を行ったところ、 MFT の有効性と few-shot learning の有用性が確認された。

![スクリーンショット 2021-03-16 20 41 28](https://user-images.githubusercontent.com/36360393/111303864-61451380-8698-11eb-9904-a5124d2594c6.png)

## Comment

BERT と比較したとき、 MLNI において数%の精度改善が確認された。
Multi-domain text mining の先行研究について触れられていない点が残念なところ。
